cmd/doctor.go

```diff
@@ -28,10 +32,34 @@ var CmdDoctor = cli.Command{
 	Usage:       "Diagnose problems",
 	Description: "A command to diagnose problems with the current Gitea instance according to the given configuration.",
 	Action:      runDoctor,
+	Flags: []cli.Flag{
+		cli.BoolFlag{
+			Name:  "list",
+			Usage: "List the available checks",
+		},
+		cli.BoolFlag{
+			Name:  "default",
+			Usage: "Run the default checks (if neither run or all is set this is the default behaviour)",
```

Comment:
```suggestion
			Usage: "Run the default checks (if neither --run or --all is set, this is the default behaviour)",
```
Using `--` makes it easier to read for non-English natives, I think. 🙄